downstream/modules/platform/ref-hub-variables.adoc

```diff
@@ -61,47 +61,40 @@ If you only want one type of content, set `automationhub_seed_collections` to `t
 | *`automationhub_collection_signing_service_script`* | If a collection signing service is enabled, you must provide this variable to ensure that collections can be properly signed.
 
 `/absolute/path/to/script/that/signs`
-| *`automationhub_create_default_collection_signing_service`* | The default install does not create a collection signing service.
-If set to `true` a signing service is created.
+| *`automationhub_create_default_collection_signing_service`* | Set this variable to true to create a collection signing service.
 
-Default = `false`
+Default = `false`.
 | *`automationhub_container_signing_service_key`* | If a container signing service is enabled, you must provide this variable to ensure that containers can be properly signed.
 
 `/absolute/path/to/key/to/sign`
 | *`automationhub_container_signing_service_script`* | If a container signing service is enabled, you must provide this variable to ensure that containers can be properly signed.
 
 `/absolute/path/to/script/that/signs`
-| *`automationhub_create_default_container_signing_service`* | The default install does not create a container signing service.
-If set to `true` a signing service is created.
+| *`automationhub_create_default_container_signing_service`* | Set this variable to true to create a container signing service.
 
-Default = `false`
-| *`automationhub_disable_hsts`* | The default install deploys a TLS enabled {HubNameMain}.
-Use if {HubName} is deployed with _HTTP Strict Transport Security_ (HSTS) web-security policy enabled.
-Unless specified otherwise, the HSTS web-security policy mechanism is enabled.
-This setting allows you to disable it if required.
+Default = `false`.
+| *`automationhub_disable_hsts`* | The default installation deploys a TLS enabled {HubNameMain}.
+Use this variable if you deploy {HubName} with _HTTP Strict Transport Security_ (HSTS) web-security policy enabled.
+This variable disables, the HSTS web-security policy mechanism.
 
-Default = `false`
+Default = `false`.
 | *`automationhub_disable_https`* | _Optional_
 
 If {HubNameMain} is deployed with HTTPS enabled.
 
 Default = `false`.
-| *`automationhub_enable_api_access_log`* | When set to `true`, creates a log file at `/var/log/galaxy_api_access.log` that logs all user actions made to the platform, including their username and IP address.
+| *`automationhub_enable_api_access_log`* | When set to `true`, this variable creates a log file at `/var/log/galaxy_api_access.log` that logs all user actions made to the platform, including their username and IP address.
 
 Default = `false`.
 | *`automationhub_enable_analytics`* | A Boolean indicating whether to enable pulp analytics for the version of pulpcore used in {HubName} in {PlatformNameShort} {PlatformVers}.
 
-To enable pulp analytics, set `automationhub_enable_analytics = true`.
+To enable pulp analytics, set `automationhub_enable_analytics to true`.
```

Comment:
Instead of 'automationhub_enable_analytics to true'
I wonder if it should be 'automationhub_enable_analytics' to true. i.e. just put the quotes around the collection name?